test/onnx/verify_onnx.cpp

```diff
@@ -1014,6 +1014,82 @@ TEST_CASE(instance_norm_3d_test)
     EXPECT(migraphx::verify::verify_rms_range(result_vector, gold));
 }
 
+TEST_CASE(isinf_half_test)
+{
+    migraphx::program p = migraphx::parse_onnx("isinf_half_test.onnx");
+    p.compile(migraphx::make_target("ref"));
+
+    migraphx::shape s{migraphx::shape::half_type, {2, 3}};
+    migraphx::parameter_map pp;
+    std::vector<float> tmp = {-1.7, NAN, INFINITY, 3.6, -INFINITY, INFINITY};
```

Comment:
Suggestion: Instead of INFINITY, please use a synthesized value. That way we can catch any run time FP results that would bypass FP-units if compiler option like --fast-math is turned on. And also add a comment why we are doing it this way.. Thanks.

example: 
float x = std::numeric_limits<float>::max();
float infinity = x * x; 
(And then use this to fill tmp)

Reply:
I think it's better to explicitly use `NAN` and `INFINITY`. The purpose is to check the isinf operator not to check if --fast-math is enabled.

Alternatively `std::numeric_limits` could be used to get these values as well.

Reply:
> I think it's better to explicitly use `NAN` and `INFINITY`. The purpose is to check the isinf operator not to check if --fast-math is enabled.


Better how? The idea is not to check if fast-math is enabled. The pain of not finding FP failure early is the issue. The idea is to fail, if FP guarantees can't me made -- this can be only found at run-time by generating infinity. Note: NAN is not an issue.  Thanks.

My suggestion is because we are specifically testing failure related to Infinity. Normally I would also use INFINITY. The proper test is to do it by generating infinity.

Reply:
Since there was no consensus on this discussion, I've added bot `std::numeric_limit` and the proposed synthesized inf values.

Reply:
Thanks.